recipes/pkgparser/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "pkgparser" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 96c10814083212a0937b9d9359fb3b35905e64dc072f65eee53718efd2e9e2c0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Looks like this is noarch and has two entrypoints.

```suggestion
  entry_points:
    - generate-package-docs=pkgparser.bin.generate_package_docs:main
    - parse-docs=pkgparser.bin.parse_docs:main
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv
  number: 0

```